python/paddle/fluid/tests/unittests/test_var_base.py

```diff
@@ -952,9 +952,21 @@ def _test_list_index(self):
         array = np.arange(120).reshape([6, 5, 4])
         x = paddle.to_tensor(array)
         py_idx = [[0, 2, 0, 1, 3], [0, 0, 1, 2, 0]]
+
+        # note(chenjianye):
+        # Non-tuple sequence for multidimensional indexing is supported in numpy < 1.24.
+        # For List case, the outermost `[]` will be treated as tuple `()` in version less than 1.24,
+        # which is used to wrap index elements for multiple axes.
+        # And from 1.24, this will be treat as a whole and only works on one axis.
+        #
+        # e.g. x[[[0],[1]]] = x[([0],[1])] = x[[0],[1]] (in version < 1.24)
+        #      x[[[0],[1]]] = x[array([[0],[1]])] (in version >= 1.24)
```

Comment:
it is better to understand that using `==` instead of `=`

Reply:
in numpy version >= 1.24, adding the description of behavior of x[tuple([[0],[1]])]), It is easier to see that such modification is working.